packages/vector_graphics_compiler/test/parser_test.dart

```diff
@@ -4,6 +4,37 @@ import 'package:vector_graphics_compiler/vector_graphics_compiler.dart';
 import 'test_svg_strings.dart';
 
 void main() {
+  test('None on fill', () {
+    const String svg = '''
+<svg xmlns="http://www.w3.org/2000/svg" width="384" height="384" fill="none"
```

Comment:
So this defaults to none, so anything that doesn't supply a fill gets none by default?

Reply:
Right - this means fills should default to nothing instead of black. So the rect below here is actually a no-op because it has no stroke specified (default for stroke is none) and the fill gets inherited as none.